transform/transformers/mbs_transformer.py

```diff
@@ -223,23 +227,49 @@ def check_turnover_totals(self):
 
             for q_id in self.turnover_questions:
                 try:
-                    turnover_totals[q_id] = self.round_mbs(self.response["data"].get(q_id))
+                    turnover_totals[q_id] = self.round_mbs(
+                        self.response["data"].get(q_id)
+                    )
                 except TypeError:
                     logger.info("No answer supplied for {}. Skipping.".format(q_id))
 
             return turnover_totals
 
+    def survey_dates(self):
+        """If questions 11 or 12 don't appear in the survey data, then populate
+        them with the period start and end date found in the metadata
+        """
+        try:
+            start_date = MBSTransformer.parse_timestamp(self.response["data"]["11"])
+        except KeyError:
+            logger.info("Populating start date using metadata")
+            start_date = datetime.datetime.strptime(
+                self.response.get("metadata", {})["ref_period_start_date"], "%Y-%m-%d"
```

Comment:
why not use MBSTransfromer.parse_timestamp here?

Reply:
Good point